GetRawYieldsDplusDs.py

```diff
@@ -17,6 +18,7 @@
 parser.add_argument('fitConfigFileName', metavar='text', default='config_Ds_Fit.yml')
 parser.add_argument('centClass', metavar='text', default='')
 parser.add_argument('inFileName', metavar='text', default='')
+parser.add_argument('--refFileName', metavar='text', default='')
```

Comment:
I would put this option at the end of the parsed "needed" files, i.e., after outFileName